mozilla/fx-private-relay
# Add link to a survey for new phone masking users

# New feature description

This adds a link to a survey to gather user input on how to iterate on phone masking.

# Screenshot (if applicable)

![image](https://user-images.githubusercontent.com/4251/199675117-0451fca1-08e1-427f-bf87-fb46ee6e65a4.png)

# How to test

With your user in the US or Canada (locally, with your browser language set to e.g. `en_US`) and language set to English, visit the website with a user that has the phone masking plan. You should see the banner.

# Checklist

- [ ] l10n changes have been submitted to the l10n repository, if any. - N/A, English only
